__tests__/components/__snapshots__/LessonCard.js.snap

```diff
@@ -51,7 +51,11 @@ exports[`Lesson Card Complete State Should render lessonCard with loading... 1`]
         >
           <h4
             class="lesson-card__title font-weight-bold mt-3"
-          />
+          >
+            <a
+              href="/curriculum/undefined"
```

Comment:
This is not supposed to be undefined right? /curriculum/undefined

Reply:
Correct, I believe this is one of the test lessons that got added and has no lessonId. But, maybe it would be a good idea to check for undefined id and do something else?